test/server/cards/01_SOR/events/ForceThrow.spec.ts

```diff
@@ -0,0 +1,113 @@
+            });
+
+            it('should select opponent to discard a card but no deal damage as there is not a force unit', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['force-throw'],
+                        groundArena: ['wampa']
+                    },
+                    player2: {
+                        hand: ['karabast', 'battlefield-marine'],
+                        groundArena: ['specforce-soldier', 'atst', 'ezra-bridger#resourceful-troublemaker']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                // Opponent discards a card no force unit in play
+                context.player1.clickCard(context.forceThrow);
+                context.player1.clickPrompt('Opponent');
+                context.player2.clickCard(context.karabast);
+
+                expect(context.karabast).toBeInZone('discard');
+                expect(context.player2).toBeActivePlayer();
+            });
+
+            it('should select myself to discard a card but no cards to discard', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['force-throw'],
+                        groundArena: ['wampa']
+                    },
+                    player2: {
+                        hand: ['karabast', 'battlefield-marine'],
+                        groundArena: ['specforce-soldier', 'atst', 'ezra-bridger#resourceful-troublemaker']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.forceThrow);
+                context.player1.clickPrompt('You');
+
+                expect(context.player2).toBeActivePlayer();
+            });
+
+            it('should select myself to discard a card and deal damage to an enemy unit', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['force-throw', 'karabast'],
+                        groundArena: ['wampa', 'ezra-bridger#resourceful-troublemaker']
+                    },
+                    player2: {
+                        groundArena: ['specforce-soldier']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                // Discard a card and no force unit in play
+                context.player1.clickCard(context.forceThrow);
+                context.player1.clickPrompt('You');
+                context.player1.clickPrompt('Deal damage to a unit equal to the cost of Karabast (2 damage)');
+                context.player1.clickCard(context.specforceSoldier);
+
+                expect(context.karabast).toBeInZone('discard');
+                expect(context.specforceSoldier).toBeInZone('discard');
```

Comment:
When we're doing a test that deals damage we should always choose a target that survives the damage, if possible, so that we can validate the amount